source/docs/FlytPi/FlytPi_Getting_Started.rst

```diff
@@ -3,10 +3,20 @@
 Getting Started with FlytPi
 ===========================
 
+|br|
+
+..  youtube:: mYnLhnN0zKk
+    :aspect: 16:9
+    :width: 80%
+
+|br|
+
+
 .. .. figure:: /_static/Images/FlytPi.png
 .. 	:align: center
-.. 	:scale: 120 %
+.. 	:scale: 50 %
```

Comment:
I don't think this image would be rendered! Are you sure you don't want it to be rendered.
If yes, why are you changing scale from 120 to 50 ?